frontend/src/lang/pl-PL.json

```
@@ -0,0 +1,94 @@
+{
+    "languageName": "Polish",
+    "Create your admin account": "Utwórz swoje konto administratora",
+    "authIncorrectCreds": "Nieprawidłowa nazwa użytkownika lub hasło.",
+    "PasswordsDoNotMatch": "Hasła nie pasują do siebie.",
+    "Repeat Password": "Powtórz hasło",
+    "Create": "Utwórz",
+    "signedInDisp": "Zalogowany jako {0}",
+    "signedInDispDisabled": "Autoryzacja wyłączona.",
+    "home": "Strona główna",
+    "console": "Konsola",
+    "registry": "Rejestr",
+    "compose": "Stwórz",
+    "addFirstStackMsg": "Stwórz swój pierwszy stos!",
+    "stackName" : "Nazwa stosu",
+    "deployStack": "Wdroż",
+    "deleteStack": "Usuń",
+    "stopStack": "Zatrzymaj",
+    "restartStack": "Uruchom ponownie",
+    "updateStack": "Aktualizuj",
+    "startStack": "Uruchom",
+    "editStack": "Edytuj",
+    "discardStack": "Odrzuć",
+    "saveStackDraft": "Zapisz",
+    "notAvailableShort" : "N/D",
+    "deleteStackMsg": "Czy na pewno chcesz usunąć ten stos?",
+    "stackNotManagedByDockgeMsg": "Ten stos nie jest zarządzany przez Dockge.",
+    "primaryHostname": "Podstawowa nazwa hosta",
+    "general": "Ogólne",
+    "container": "Kontener | Kontenery",
+    "scanFolder": "Skanuj folder ze stosami",
+    "dockerImage": "Obraz",
+    "restartPolicyUnlessStopped": "Chyba że zatrzymano",
+    "restartPolicyAlways": "Zawsze",
+    "restartPolicyOnFailure": "Przy awarii",
+    "restartPolicyNo": "Nie",
+    "environmentVariable": "Zmienna środowiskowa | Zmienne środowiskowe",
+    "restartPolicy": "Polityka restartu",
+    "containerName": "Nazwa kontenera",
+    "port": "Port | Porty",
+    "volume": "Wolumen | Wolumeny",
+    "network": "Sieć | Sieci",
+    "dependsOn": "Zależność kontenera | Zależności kontenera",
+    "addListItem": "Dodaj {0}",
+    "deleteContainer": "Usuń",
```

```suggestion
    "deleteContainer": "Usuń kontener",
```
consistency with translations below